cligj/features.py

```diff
@@ -115,6 +129,26 @@ def iter_features(geojsonfile, func=None):
                     newfeat = func(feat)
                     if newfeat:
                         yield newfeat
+            elif 'coordinates' in obj:
+                newfeat = func(to_feature(obj))
+                if newfeat:
+                    yield newfeat
+
+
+def to_feature(obj):
```

Comment:
Could we punt on that and cover that use case with a more general tool like the [proposed `fio calc`](https://github.com/Toblerity/Fiona/issues/273)?